x-pack/test/security_solution_api_integration/test_suites/detections_response/utils/alert/get_query_alerts_ids.ts

```diff
@@ -0,0 +1,23 @@
+/*
```

Comment:
I initiated the relocation process because I'm currently reviewing the files to assess whether any renaming is necessary, for, ex: all files and functions that originally referenced "Alerts" as "Signals" have been renamed, I think it will be easier to move them gradually to be able to add them to the relevant folder as well as go through any needed refactor in chunks so they will be easily reviewed and tested

Reply:
Maybe we can keep it at the same location and when everything migrated, move them to a new one?Otherwise, keeping 2 versions will lead to high maintenance cost(diverging one version from another): keeping both version in sync might be difficult, especially, if it is not known whether util has duplicate or not